iris_grib/tests/unit/grib_phenom_translation/test_grib_phenom_translation.py

```diff
@@ -31,11 +31,12 @@
 import cf_units
 
 import iris_grib.grib_phenom_translation as gptx
+from iris_grib.grib_phenom_translation import _LookupTable
```

Comment:
Or you could keep the `from ... import` style and add the only other import (`grib1_phenom_to_cf_info`) to the import line.